src/main/native/cpp/RegisterIOMau.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * RegisterIO.h
+ *
+ *  Created on: Jul 29, 2015
+ *      Author: Scott
+ */
+
+#ifndef SRC_REGISTERIOMAU_H_
+#define SRC_REGISTERIOMAU_H_
+
+#include "IIOProvider.h"
+#include "IRegisterIO.h"
+#include "IMUProtocol.h"
+#include "AHRSProtocol.h"
+#include "IBoardCapabilities.h"
+#include "IIOCompleteNotification.h"
+
+#include <stdint.h>
+
+class RegisterIOMau : public IIOProvider
```

Comment:
Remove this whole file. It provides support for VMXpi which we aren't setup for currently.

Reply:
Just curious, in what way are we not currently setup for it?